README.md

```diff
@@ -71,14 +74,35 @@ services:
     supervisor:
         image: 31z4/storm:1.0.2
         container_name: supervisor
-        command: storm supervisor
+        command: >
+            storm supervisor
+            -c storm.zookeeper.servers="[\"zookeeper\"]"
```

Comment:
This is done by default if no config provided. Please see https://github.com/31z4/storm-docker/blob/master/1.0.2/docker-entrypoint.sh#L15-L16